runtime/packages/modules/timestamp/assembly/timestamp.ts

```diff
@@ -54,36 +52,36 @@ export class Timestamp{
      * it must be greater than the last one (set into storage) with at least a MinimumPeriod
      * @param now timestamp number
      */
-    static set(now: u64): void {
+    static set(now: u64): u8[] {
         const didUpdate = Storage.get(Timestamp.SCALE_TIMESTAMP_DID_UPDATE);
         const didUpdateValue: Bool = didUpdate.isSome() ? Bool.fromU8a((<ByteArray>didUpdate.unwrap()).values) : new Bool(false);
         if(didUpdateValue.value){
             Log.error('Validation error: Timestamp must be updated only once in the block');
-            throw new Error();
+            return ResponseCodes.dispatchError(1, 1);
```

Comment:
I suppose that this `dispatchError` can be called for any module in the Runtime. This is the Module-specific errors, right?
If that is the case:
1. can't we set the `moduleIndex` to a constant somewhere in the file and the `errorValue` (which I suppose is the error code to a variable as-well. It will remove the magic numbers that we are setting at the moment
2. Even better would be if we have functions in the `Timestamp` module f.e `tooFrequentSetResponseCode` or `timeframeTooLowResponseCode` (example)

My main concern is having those magic 1,1 and 1,2 numbers